src/cgls/cgls.c

```diff
@@ -224,7 +224,7 @@ static int run(int argc, char *argv[]) {
                                         RuntimeScope scope = arg_show_unit == SHOW_UNIT_USER ? RUNTIME_SCOPE_USER : RUNTIME_SCOPE_SYSTEM;
 
                                         /* Connect to the bus only if necessary */
-                                        r = bus_connect_transport_systemd(BUS_TRANSPORT_LOCAL, NULL, scope, &bus);
+                                        r = bus_connect_transport(BUS_TRANSPORT_LOCAL, NULL, scope, &bus);
```

Comment:
uh, hm, i am not conviced. I thinks systemd-cgls is tremendously useful when debugging early boot/initrd issues.

I think it's very important it works in that scenario